src/xorIt.js

```diff
@@ -10,23 +6,19 @@
  * http://cwestblog.com/2013/08/19/javascript-simple-vernam-cipher/
  */
 
-var xorIt = (key, textOrCipher) => {
+export default function xorIt(key, textOrCipher) {
   if (!key || !textOrCipher) {
     throw 'XorIt: needs key and textOrCipher';
   }
 
-  let cipher = [],
-      keyLength = key.length,
-      fromCharCode = String.fromCharCode,
-      len = textOrCipher.length,
-      i = 0;
+  let cipher = [];
```

Comment:
eslint complains about this let and says it should be 'const'. I changed it to const and ran the tests and everything was okay. Does it makes sense to change it to const?